### PR TITLE
[PL/HT] Move the permeability specification from solid phase to medium scale.

### PR DESCRIPTION
**Pull request by TomFischer**
Used the following bash script for changing project files:
```
#!/usr/bin/env bash

number_of_medium_tags=`xmlstarlet sel -t -v "count(///*/medium)" $1`
number_of_medium_tags=$(($number_of_medium_tags - 1))

xmlstarlet ed --move "//OpenGeoSysProject/media/medium[@id=0]/phases/phase/properties/property[name='permeability']" "//OpenGeoSysProject/media/medium[@id=0]/properties" $1 > /tmp/t0.prj

for i in `seq 1 ${number_of_medium_tags}`;
do
    i_m_e=$(($i-1))
    #echo "${i}, ${i_m_e}"
    path="//OpenGeoSysProject/media/medium[@id="${i}"]"
    xmlstarlet ed --move "$path/phases/phase/properties/property[name='permeability']" "$path/properties" /tmp/t$i_m_e.prj > /tmp/t${i}.prj
done

#format the result file
xmlstarlet fo -s 4 /tmp/t${number_of_medium_tags}.prj > ${1}
```

### CODE REVIEW

File: ProcessLib/HT/HTFEM.h [hunk 1]
```diff
@@ -294,8 +293,7 @@ class HTFEM : public HTLocalAssemblerInterface
                 MaterialPropertyLib::Variable::phase_pressure)] = p_int_pt;
 
             auto const K = MaterialPropertyLib::formEigenTensor<GlobalDim>(
-                solid_phase
```
Comment (endJunction) [inline ProcessLib/HT/HTFEM.h]: `solid_phase` is now unused in line 275

**Comment by renchao-lu** [comment]
`Porosity` would be better also set as of medium property.

**Comment by TomFischer** [comment]
> `Porosity` would be better also set as of medium property.

I'll prepare another PR for the changes.

File: ProcessLib/HT/HTFEM.h (hunk 1 above)
Comment (endJunction) [reply ProcessLib/HT/HTFEM.h]: bump

**Pull request merged**